utils/rpms/daos.spec

```diff
@@ -110,6 +110,7 @@ BuildRequires: libcurl4
 %endif
 %endif
 %endif
+BuildRequires: libuuid-devel
```

Comment:
this change was not part of the original PR. is it carried over by accident somehow?

Reply:
ah i see the issue. it probably is better to cherry pick the entire commit from Brian:
https://github.com/daos-stack/daos/pull/11361

Reply:
Thank you! Just did cherry pick [2df58d1](https://github.com/daos-stack/daos/commit/2df58d1e2d41075757b868a7c50376aaa7c49f24)

Reply:
Hrm.  I already have a PR in flight with the changes I have made to master that need backporting to this branch.  I guess these are going to conflict now.

Reply:
@brianjmurrell Thank you! Then I will wait for your PR landed then undo [2df58d1](https://github.com/daos-stack/daos/commit/2df58d1e2d41075757b868a7c50376aaa7c49f24) in this PR.

Reply:
@brianjmurrell Which PR were you referring to? I can refresh this one after yours is landed. Thank you!

Reply:
#11519